app/assets/stylesheets/core/_elements.scss

```diff
@@ -6,14 +6,3 @@ html {
 ul li:before {
   display: none;
 }
-
```

Comment:
Do we need these base elements to have their sizes defined? Will this affect layouts across the app?